README.md

```diff
@@ -10,9 +10,9 @@ This is the repository for Vue 3.0.
 
   ```bash
   # npm
-  npm init @vitejs/app
+  npm init vite
```

Comment:
I think it makes more sense to include the instructions of vite:

```bash
# npm 6.x
npm init vite@latest my-vue-app --template vue

# npm 7+, extra double-dash is needed:
npm init vite@latest my-vue-app -- --template vue

# yarn
yarn create vite my-vue-app --template vue
```

Reply:
oh,yes. npm@7 has that beaking change, and we should always pay attention